src/wasm/wasm-s-parser.cpp

```diff
@@ -362,7 +362,8 @@ SExpressionWasmBuilder::SExpressionWasmBuilder(Module& wasm,
         stringToBinary(str, size, data);
       }
     }
-    WasmBinaryBuilder binaryBuilder(wasm, data);
+    // TODO: support applying features here
+    WasmBinaryBuilder binaryBuilder(wasm, FeatureSet::MVP, data);
```

Comment:
```suggestion
    WasmBinaryBuilder binaryBuilder(wasm, FeatureSet::All, data);
```
We should leave disallowed features to be caught by validation.

Reply:
I don't understand you here either, sorry... How would enabling all features get us more validation in the text parser? (the text parser has no feature section parsing, but even if it did..?)

(tests passed)

Reply:
I was thinking that we would want to be able to parse all features by default, but the feature set passed here won't actually control what features can be parsed, so I was confused. Disregard this.

Reply:
Actually, why does this need to change to apply a new feature set? Can't it just keep the default one that `wasm` already had?

Reply:
(same issue as above, I think)